packages/material-ui/src/NativeSelect/NativeSelectInput.js

```diff
@@ -77,7 +77,6 @@ const NativeSelectRoot = experimentalStyled(
 
       return {
         ...styles.root,
-        ...styles.select,
```

Comment:
Shouldn't it be the opposite?

Reply:
From what I understand, we have this DOM structure:

- SelectRoot - InputBaseRoot
   - SelectSelect
   - SelectIcon
   - etc.